kover-gradle-plugin/examples/android/dynamic/build.gradle.kts

```diff
@@ -0,0 +1,6 @@
+plugins {
+    id("com.android.application") version "7.4.0" apply false
+    id("com.android.library") version "7.4.0" apply false
```

Comment:
Shouldn't `"com.android.dynamic-feature"` also be here with version?

Reply:
I think this version got implicitly together with other plugins, so it did not lead to errors. But for clarity I will add